### PR TITLE
New package: vit 2.1.0

### PR DESCRIPTION
**Pull request by Pinoinha**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/vit/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'vit'
+pkgname=vit
+version=2.1.0
+revision=0
```
Comment (paper42) [inline srcpkgs/vit/template:4]:
```suggestion
revision=1
```

File: srcpkgs/vit/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+build_style=python3-module
+hostmakedepends="python3-pip task python3-pytz python3-urwid python3-tzlocal"
+depends="task"
+checkdepends=
```
Comment (paper42) [inline srcpkgs/vit/template:8]:
```suggestion
hostmakedepends="python3-setuptools"
depends="task python3-pytz python3-urwid python3-tzlocal"
```

File: srcpkgs/vit/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+maintainer="pinoinha <pinoinha@disroot.org"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+changelog="https://raw.githubusercontent.com/vit-project/vit/2.x/CHANGES.md"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=56676651164fb71878a0c1e4bb369cfa336d46e6fd0de7e6167b57fca7eaf662
+
+pre_build() {
+	pip3 install tasklib
+}
```
Comment (paper42) [inline srcpkgs/vit/template:19]: Why is this necessary? It's not acceptable to run pip in package templates, if we need to package tasklib, it should be done in a separate commit in this PR.

File: srcpkgs/vit/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+build_style=python3-module
+hostmakedepends="python3-pip task python3-pytz python3-urwid python3-tzlocal"
+depends="task"
+checkdepends=
+short_desc="VIT is a lightweight, fast, curses-based front end to Taskwarrior "
+maintainer="pinoinha <pinoinha@disroot.org"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+changelog="https://raw.githubusercontent.com/vit-project/vit/2.x/CHANGES.md"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/vit/template:14]:
```suggestion
distfiles="https://github.com/vit-project/vit/archive/refs/tags/v${version}.tar.gz"
```

**Review by paper42 — CHANGES_REQUESTED** [review]

File: srcpkgs/vit/template [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'vit'
+pkgname=vit
+version=2.1.0
+revision=0
+build_style=python3-module
+hostmakedepends="python3-pip task python3-pytz python3-urwid python3-tzlocal"
+depends="task"
+checkdepends=
+short_desc="VIT is a lightweight, fast, curses-based front end to Taskwarrior "
+maintainer="pinoinha <pinoinha@disroot.org"
```
Comment (paper42) [inline srcpkgs/vit/template:10]:
```suggestion
maintainer="pinoinha <pinoinha@disroot.org>"
```

File: srcpkgs/vit/template (hunk 3 above)
Comment (Pinoinha) [reply srcpkgs/vit/template:18]: that's a good point. I'll package it and reopen this PR afterwards

**Comment by Pinoinha** [comment]
as suggested by @paper42 I'll package tasklib beforehand, to prevent the pre_build() pip installation. this PR will be reopened afterwards.

**Comment by abenson** [comment]
Leave it open and mark it draft until its ready.

File: srcpkgs/vit/template (hunk 3 above)
Comment (paper42) [reply srcpkgs/vit/template:18]: You don't need to close this PR

**Comment by Pinoinha** [comment]
added the python3-tasklib template file to this same PR, as suggested by @ericonr 

also changed the files as recommended by @paper42

**Comment by paper42** [comment]
please fix the CI - the lint step runs xlint (from xtools) on the templates and the other checks are from tests, so add necessary packages to checkdepends.

This PR should contain 2 commits, one for each new package.

**Comment by Barbarossa93** [comment]
This can be closed due to https://github.com/void-linux/void-packages/pull/35016
